src/languages/en.js

```diff
@@ -105,6 +105,7 @@ export default {
         leaveRoom: 'Leave room',
         your: 'your',
         conciergeHelp: 'Please reach out to Concierge for help.',
+        maxParticipantsReached: ({count}) => `You've selected the maximum number (${count}) of participants.`,
```

Comment:
NAB: But any specific reason we moved it as `common.maxParticipantsReached` instead of `messages.maxParticipantsReached` ?

Reply:
All the other texts that are commonly used are in `common` object, so it made sense to add this one to `common` as well.